api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -64,12 +69,32 @@ public Profile getProfile(User user) throws ApiException {
       profile.setDemographicSurveyCompletionTime(user.getDemographicSurveyCompletionTime()
           .getTime());
     }
+    // TODO: Add call to mailchimp using user email hash from DB to get email verified status.
     if (user.getDataAccessLevel() != null) {
       profile.setDataAccessLevel(user.getDataAccessLevel());
     }
     if (user.getAuthorities() != null) {
       profile.setAuthorities(new ArrayList(user.getAuthorities()));
     }
+    String userEmailVerificationStatus = null;
+    if(user.getContactEmail() != null) {
+      try {
+        userEmailVerificationStatus = mailChimpService.getMember(user.getContactEmail());
```

Comment:
I don't love having to call this external service on every call to getProfile()... once we get back a status of subscribed, can we cache that on User so we don't have to call this anymore?